homeassistant/components/motion_blinds/gateway.py

```diff
@@ -12,6 +12,13 @@
 _LOGGER = logging.getLogger(__name__)
 
 
+def device_name(blind):
```

Comment:
Either add this file to coveragerc, or preferably, write a simple test for this function.

Reply:
I just added a test that schould bring the coverage back to 100%